### ISSUE TITLE
Anywhere to find a replacement for this package?

### ISSUE THREAD

**Issue by bisraelsen** [johnmyleswhite/PowerAnalysis.jl#1]
I see that this package is no longer maintained. Is similar functionality found in another julia package, or should I try to get this working?

**Comment by johnmyleswhite** [comment]
I don't this package was ever publicly released. I may release it some day, but I would recommend looking elsewhere.

**Comment by bisraelsen** [comment]
You're right. Maintained implies having been released, which this wasn't. Off the top of your head, are you aware of a package that might have this capability?

**Comment by casasgomezuribarri** [comment]
I was wondering if this package was expected to be out soon. 

```julia
]add PowerAnalysis
ERROR: The following package names could not be resolved:
 * PowerAnalysis (not found in project, manifest or registry)
```

**Comment by kescobo** [comment]
I had intended to release it once it got moved to JuliaStats org, but that stalled out. I'll put it back on my list

**Comment by casasgomezuribarri** [comment]
Didn't mean to laugh-react! Thanks for working on this :)

**Comment by rikhuijzer** [comment]
I'm in the process of registering [PowerAnalyses.jl](https://github.com/rikhuijzer/PowerAnalyses.jl). I've thought for a long time about the API and decided to not base it on the `HypothesisTests.jl` types like this package does. Also, I've chosen a different name than `PowerAnalysis.jl` so that it would still be possible to register this package if that's desired :)

Feedback on PowerAnalyses.jl is welcome. The package is far from "done", but some basic power analyses work and `get_es` and `get_n` are actually more accurate than `pwr` and `G*Power`. So that's why I thought it was already a good idea to register the package
